sympy/polys/multivariate_resultants.py

```diff
@@ -127,37 +132,58 @@ def get_dixon_polynomial(self):
         return poly_from_expr(dixon_polynomial, self.dummy_variables)[0]
 
     def get_upper_degree(self):
+        SymPyDeprecationWarning(feature="get_upper_degree",
+                        useinstead="get_max_degrees",
+                        issue=17763,
+                        deprecated_since_version="1.5").warn()
+
         list_of_products = [self.variables[i] ** self.max_degrees[i]
```

Comment:
I now get two warnings from `get_upper_degree`, including one for `max_degrees`. This line could have `self._max_degrees[i]` to avoid the second warning.

Reply:
I see. I didn't notice because I think the warnings are not displayed correctly to me.
Is that how the warning is supposed to look like?
```
get_upper_degree has been deprecated since SymPy 1.5. Use
get_max_degrees instead. See
https://github.com/sympy/sympy/issues/17763 for more info.

  deprecated_since_version="1.5").warn()
3
```

Reply:
I think that the first three lines are the actual message. The last line is something that the interpreter will add. It is probably not useful but not harmful either.